docs/scripts/prtvol2csv.rst

```diff
@@ -39,8 +39,8 @@ Additionally, if the Eclipse DATA file includes::
   RPTSOL
     FIP=2 'FIPRESV' /
 
-the PRT-file will also contain a table with pore volumes pr. phase and pr.
-FIPNUM, which will be added to the exported table.
+the PRT-file will also contain a table with pore volumes pr. phase and pr. FIPNUM, 
```

Comment:
```suggestion
The PRT-file will also contain a table with pore volumes pr. phase and pr. FIPNUM, 
```

maybe uppercase this while here

Reply:
l. 42 is probably intended to be next part of the sentence on line 37, so maybe better to remove the ":" on line 37?  (Upper case after ":")  ("Additionally, if the Eclipse DATA file includes"  < box> "the PRT-file will also ..."  I will try to change like that.